buildinfo_test.go

```diff
@@ -658,6 +658,73 @@ func TestBuildAddGitEnvBuildNameAndNumber(t *testing.T) {
 	testBuildAddGit(t, true)
 }
 
+func TestBuildPublishWithOverwrite(t *testing.T) {
+	initArtifactoryTest(t, "")
+	buildName := "overwrite-test-build"
```

Comment:
I suggest using `tests.RtBuildName1` as the build number. The test suite initializes it with a unique suffix (OS & timestamp) to avoid collisions with parallel test runs.